doc/source/cost.rst

```diff
@@ -48,35 +83,14 @@ you.
    <a target="_blank" href="http://mybinder.org/v2/gh/jupyterhub/zero-to-jupyterhub-k8s/master?filepath=doc/ntbk/draw_function.ipynb">
    <button style="background-color: rgb(235, 119, 55); border: 1px solid; border-color: black; color: white; padding: 15px 32px; text-align: center; text-decoration: none; font-size: 16px; margin: 4px 2px; cursor: pointer; border-radius: 8px;">Launch the Cost Estimator</button></a>
 
-For a description of how these costs are broken down, see below.
-
-Users
------
-
-The number of users has a direct relationship to cost estimates. Since a
-deployment may support different types of users (i.e. researchers, students,
-instructors) with varying hardware and storage needs, take into account both the
-type of users and the number per type.
-
-User usage patterns
--------------------
-
-Another important factor is what usage pattern your users will have. Will they
-all use the JupyterHub at once, such as during a large class workshop?
-will users use JupyterHub at different times of day?
-
-The usage patterns and peak load on the system have important implications for
-the resources you need to provide. In the future JupyterHub will have
-auto-scaling functionality, but currently it does not. This means that you need
-to provision resources for the *maximum* expected number of users at one time.
-
 
 Examples
```

Comment:
Maybe we move this further to the top (or reference it from the beginning) when we have a few more examples. It probably provides the best examples/guidance on how to work out how much it might cost you. As the hard part is converting "N students in my semester long course" into "number of CPUs and RAM to have available". After that you "just " have to look up your cloud providers prices ;)